tracer/test/Datadog.Trace.TestHelpers/SpanMetadataRules.cs

```diff
@@ -231,12 +231,13 @@ public static Result IsKafka(this MockSpan span) => Result.FromSpan(span)
             .Properties(s => s
                 .MatchesOneOf(Name, "kafka.consume", "kafka.produce")
                 .Matches(Type, "queue"))
+            .Metrics(s => s
```

Comment:
Similar to "profiler" in the other PR, I really wish we would stop calling these "metrics." Terrible misnomer. There are string tags and numeric tags.

Reply:
Except... They're added to the "metrics" section, and we add them by calling `SetMetric` and.... 😛  You're fighting against the tide here 😂

Reply:
IMO, in this case, it's better to keep the "metric" nomenclature and maintain internal consistency at least.

Reply:
boo ok 😞